Javascript-tutorial/Shopping-site/index.js

```diff
@@ -0,0 +1,8 @@
+
+let errorParagraph = document.getElementById("error") 
```

Comment:
does this work when you have given multiple elements with same id's

Reply:
you can use call elements by their class name or can use the queryselector all method if you need to call multiple elements with same class names, you will come accross this on going forward with the tutorial